### ISSUE TITLE
"Failed to write report" when "verbose" is set to true

### ISSUE THREAD

**Issue by berkon** [Namchee/conventional-pr#101]
When "verbose" is set to true, conventional-pr fails to write the report to the PR. Response is "422 Invalid request". The response from GitHub API says:

```
No subschema in "oneOf" matched.
"commit_id", "path", "position" weren't supplied.
"in_reply_to" wasn't supplied.
"commit_id", "line", "path" weren't supplied.
"commit_id", "path", "subject_type" weren't supplied. []
```

![image](https://github.com/Namchee/conventional-pr/assets/28842661/ee841643-fa2a-4112-900b-7e6c1bfe6dcb)

**Comment by Namchee** [comment]
Thanks for filing an issue!

I can reproduce this, seems like the GQL migration isn't going on smoothly. I'll fix this ASAP

**Comment by Namchee** [comment]
Apparently, this is caused by wrong API endpoint when creating a pull request report. Will be fixed in next patch version.

**Comment by Namchee** [comment]
I've released `v0.15.2` that fixes this issue as evidenced by [this report](https://github.com/Namchee/conventional-pr/pull/104#issuecomment-1863799767). If you still have issues regarding verbose reporting, feel free to re-open this issue.

@all-contributors, please add @berkon for bug reporting

**Comment by berkon** [comment]
Thanks a lot Christopher for your very quick reaction! This really helps us a lot!

Regards,
Bernd
